python/ngen_conf/tests/data/test_config.json

```diff
@@ -6,7 +6,7 @@
                 "name": "bmi_multi", 
                 "params": {
                     "name": "bmi_multi", 
-                    "model_type_name": "NoahOWP_CFE", 
+                    "model_type_name": "BMIMulti", 
```

Comment:
Should we revert this if "BMIMulti" isn't the required literal now?

Reply:
That's fine with me. Good catch